app/facades/forecast_facade.rb

```diff
@@ -0,0 +1,36 @@
+class ForecastFacade
+  attr_reader :lat,
+              :lon
+
+  def initialize(lat, lon)
+    @lat = lat
+    @lon = lon
+    @service = WeatherService.new
+  end
+
+  def response_forecast
+    @service.forecast(@lat, @lon)
+  end
+
+  def current_forecast
```

Comment:
Split up each type of forecast (current hourly and daily) in order to turn them into objects properly.